presto-parquet/src/main/java/io/prestosql/parquet/writer/valuewriter/TimeMicrosValueWriter.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.parquet.writer.valuewriter;
+
+import io.prestosql.spi.block.Block;
+import io.prestosql.spi.type.Timestamps;
+import io.prestosql.spi.type.Type;
+import org.apache.parquet.column.values.ValuesWriter;
+import org.apache.parquet.schema.PrimitiveType;
+
+import static java.util.Objects.requireNonNull;
+
+public class TimeMicrosValueWriter
+        extends PrimitiveValueWriter
+{
+    private final Type type;
+
+    public TimeMicrosValueWriter(ValuesWriter valuesWriter, Type type, PrimitiveType parquetType)
+    {
+        super(parquetType, valuesWriter);
+        this.type = requireNonNull(type, "type is null");
+    }
+
+    @Override
+    public void write(Block block)
+    {
+        for (int i = 0; i < block.getPositionCount(); i++) {
+            if (!block.isNull(i)) {
+                long value = type.getLong(block, i);
+                long scaledValue = value / Timestamps.PICOSECONDS_PER_MICROSECOND;
```

Comment:
Static import

We could inline this above
```java
long value = type.getLong(block, i) / PICOSECONDS_PER_MICROSECOND;

Reply:
Inlined.